### ISSUE TITLE
Improve and clean-up focus outlines on controls

### ISSUE THREAD

**Issue by ebruchez** [orbeon/orbeon-forms#5525]
Related to #5367.

We have a custom outline/highlight on focused fields, but there are some inconsistencies, in particular with Chrome.

- [x] Handwritten Signature
- [x] buttons
- [x] link buttons
- [x] groups with `tabindex="0"`
- [x] calendar icon in date field
    - missing left highlight
- [x] group around Dropdown Date
- [x] group around Fields Date
- [x] Dropdown with Search
- [x] checkboxes group
    - width 100%
- [x] radio buttons group
    - width 100%
- [x] checkbox
- [x] radio button
- [x] Image Attachment has no group
- [ ] View mode: fields are not focusable
    - except focus goes on some `role="group"` of some XBL components
    - see #5570
- [x] Scrollable Checkboxes
    - no `tabindex` on group
- [x] Select Files
- [x] "Control Settings" -> "Email Options" highlight truncated at bottom

**Comment by ebruchez** [comment]
For static readonly fields, we need to do something similar to labeled `xf:output`. Make sure the following works:

- [x] Text Field
- [ ] Text Area
- [ ] Formatted Text
- [ ] Calculated Value
- [ ] Dropdown
- [ ] Checkboxes/Radio Buttons
- [x] Attachments
    - work as is

**Comment by ebruchez** [comment]
For the Formatted Text, ideally we'd like #5366. However, this requires #1573, which cannot be done now. So we are now trying a workaround suggested by @avernet: use a `<xf:switch xxf:update="full">`.

**Comment by ebruchez** [comment]
Like for #5367, a lot was done already and that can ship with 2022.1. Enter a new issue to track remaining items.

**Comment by ebruchez** [comment]
Round 2 covered by #5570.
